### PR TITLE
Add example systemd service

### PR DESCRIPTION
**Pull request by Cretezy**

### CODE REVIEW

**Comment by donatj** [comment]
Thank you so much for the contribution! I don't personally know much at all about Systemd so I'm going to run this by a friend who does. Do you think this should be linked or directly in the README?

File: hookah.service [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Hookah
+
+[Service]
+ExecStart=/bin/bash -c "$GOPATH/bin/hookah --server-root $HOME/hooks --http-port 8808"
```
Comment (donatj) [inline hookah.service:5]: As I don't wish to make Hookah specific to Go users, could we change the example binary path to `/usr/local/bin/hookah`.  My friend wasn't 100% certain $HOME would work from systemd like this? Can you validate this? I think my preference either way would be to also change the server root to maybe `/root/hooks`?

Thoughts?

**Review by donatj — CHANGES_REQUESTED** [review]

**Comment by Cretezy** [comment]
$HOME works for user services, which is what hookah should be ran as. In theory we shouldn't run it as root for security.

**Comment by donatj** [comment]
I'm closing this. If you're still interested you could throw it in the Wiki - https://github.com/donatj/hookah/wiki

**Pull request closed without merging**